cognite/client/_api/datapoints.py

```diff
@@ -322,6 +322,55 @@ def insert_multiple(self, datapoints: List[Dict[str, Union[str, int, List]]]) ->
         dps_poster = DatapointsPoster(self)
         dps_poster.insert(datapoints)
 
+    @overload
+    def delete_point(self, point: Dict[str, Union[int, str, datetime]]) -> None:
+        ...
+
+    @overload
+    def delete_point(self, point: Sequence[Dict[str, Union[int, str, datetime]]]) -> None:
+        ...
+
+    def delete_point(
+        self, point: Union[Dict[str, Union[int, str, datetime]], Sequence[Dict[str, Union[int, str, datetime]]]]
+    ) -> None:
+        """Delete a single datapoint from one or more time series.
```

Comment:
Just a suggestion as the current sound like "delete t=200 from ts=a,b and c":
```suggestion
        """Delete individual datapoints from one or more time series.
```